src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -33,41 +33,52 @@ void P036_data_struct::reset() {
 
 # ifdef P036_FONT_CALC_LOG
 const __FlashStringHelper tFontSettings::FontName() const {
-  if (fontData == ArialMT_Plain_24)
+  if (fontData == ArialMT_Plain_24) {
```

Comment:
Why not a switch statement here?

Reply:
I will consider it.

Reply:
A switch statement gives following error:
`expression must have integral or enum type`

Reply:
> A switch statement gives following error: `expression must have integral or enum type`

OK, then leave it as it is... Now we also know why it isn't a switch statement ;)

Reply:
I kept the `if endif` construction in the function FontName()
See commit `FIX: wrong alignment (only centered) for long, non-scrolling lines`